cleverpush/src/main/java/com/cleverpush/util/VersionComparator.java

```diff
@@ -25,13 +25,13 @@ public int compare(Object o1, Object o2) {
                     suffix1 = tokenizer1.getSuffix();
                     if (number1 != 0 || suffix1.length() != 0) {
                         // Version one is longer than number two, and non-zero
-                        return 1;
+                        return VersionComparison.GREATER_THAN;
                     }
                 }
                 while (tokenizer1.MoveNext());
 
                 // Version one is longer than version two, but zero
-                return 0;
+                return VersionComparison.EQUALS;
```

Comment:
not sure if Equals is correct here, can you give some examples here?

Reply:
here if we get the 0 from this method it means that both the versions are Equal.